ts/types/IgeEngine.d.ts

```diff
@@ -4,11 +4,11 @@ declare class IgeEngine extends IgeClass {
 	_tickStart: number;
 	_renderLatency: number;
 
-    isClient: boolean;
-    isServer: boolean;
+	isClient: boolean;
+	isServer: boolean;
 
-    client: Client;
-    server: Client;
+	client: Client;
+	server: Client;
```

Comment:
I'm hung up on this. What case(s) is there `ige.server` that exists and is of type `Client`?

Reply:
`ige.server` is always assigned here: https://github.com/moddio/taro/blob/master/src/index.js#L11
notice that `App` is not a type but rather name of the argument passed to `init` method.
and only place where it is instantiated is here: https://github.com/moddio/taro/blob/master/src/index.js#L19
and is passed `Client` type. So, yes, `ige.server` is always of type `Client`. Very unfortunate naming, I agree.